recipes/unitytest/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.8)
+project(test_package LANGUAGES C)
+
+find_package(UnityTest REQUIRED CONFIG)
+
+add_executable(test_package test_package.c)
+target_link_libraries(test_package PRIVATE UnityTest::UnityTest)
```

```suggestion
target_link_libraries(test_package PRIVATE UnityTest::UnityTest)
target_compile_features(${PROJECT_NAME} PRIVATE c_std_11)
```

Need to keep the standard aligned, otherwise it will fail with older compiler in the CI: 
https://github.com/ThrowTheSwitch/Unity/blob/v2.5.2/CMakeLists.txt#L93